packages/quantic/force-app/main/default/lwc/quanticTimeframe/quanticTimeframe.js

```diff
@@ -0,0 +1,38 @@
+import { api, LightningElement} from 'lwc';
```

Comment:
If I try to generate the JSON documentation from this branch, I see the `quanticTimeframeFacet` component, but not this `quanticTimeFrame` one. Is this on purpose?

Reply:
I guess that was because the header comment was missing. `QuanticTimeframe` now appear in the generated `quantic-docs.json`.